src/main/java/uk/gov/justice/hmpps/prison/api/resource/MovementResource.java

```diff
@@ -89,9 +96,12 @@ public List<RollCount> getRollcount(@PathVariable("agencyId") @Parameter(descrip
         @ApiResponse(responseCode = "404", description = "Requested resource not found.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
         @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})})
     @Operation(summary = "Rollcount movement numbers.", description = "Rollcount movement numbers.")
+    @VerifyAgencyAccess(overrideRoles = {"ESTABLISHMENT_ROLL"})
```

Comment:
Just checking that all clients (if they had SYSTEM_USER, now have the alternative role set for each endpoint)